pkg/runs/runNameChecker.go

```diff
@@ -16,24 +16,24 @@ var (
 	// ^ matches the start of the string
 	// $ matches the end of the string
 	// If ^ and $ are not specified, then M2M2 would match (two instances of the match)!
-	RUN_NAME_PATTERN *regexp.Regexp = regexp.MustCompile("^[a-zA-Z]+[0-9]+$")
+	FLAG_NAME_PATTERN *regexp.Regexp = regexp.MustCompile("^[a-zA-Z]+[0-9]+$")
```

Comment:
Run names might have different constraints to other fields like groups, so it might be better to create a separate validator for groups.

For example, I can have a run name of "C123" and a group called "1. my monday group!" - this validator will fail for the group name because it expects a letter followed by numbers.

Some more unit tests might be needed to exercise the validation logic better.